src/averageAges.js

```diff
@@ -36,8 +43,21 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+function calculateWomenAverageAge(people, withChildren = false) {
+  let woman = people.filter(person => person.sex === 'f');
+
+  if (withChildren !== false) {
```

Comment:
```suggestion
  if (withChildren) {
```

Reply:
it's already  boolean value

Reply:
(but don't forget to move it as condition for ternary operator in filter and remove `if`)